src/test/java/com/jetbulb/interviews/NotationConverterTest.java

```diff
@@ -0,0 +1,50 @@
+package com.jetbulb.interviews;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowableOfType;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+public class NotationConverterTest {
+    private final NotationConverter converter = new NotationConverter();
+
+    @ParameterizedTest
+    @NullAndEmptySource
+    @ValueSource(strings = {"A", "1", "X1", "_", "Vi1"})
+    void shouldThrowInvalidValueExceptionIfNonRomanDigitFound(String in) {
+        assertThat(
+                catchThrowableOfType(() -> converter.toArabic(in), InvalidValueException.class)
+        )
+                .overridingErrorMessage("InvalidValueException should be thrown if invalid symbols detected")
+                .isNotNull()
+                .hasMessage("String must contain only valid roman numerals [I, V, X, L, C, D, M]");
```

Comment:
I do not think that checking messages is a good idea.
It leads us for many changes in the code if the message changed.
This just makes test fragile.